src/main/java/cz/smarteon/loxone/Codec.java

```diff
@@ -134,10 +135,12 @@ public static String writeMessage(final Object message) throws IOException {
         return JsonMapperHolder.mapper.writeValueAsString(message);
     }
 
+    @SuppressWarnings("unused") //TODO is this method still needed?
```

Comment:
what is the point of supressing unused public method? It can be used well in public...

On the other hand it's probably not and the only reason to keep it is the formal backward compatibility. So let's rather mark it as `Deprecated` only.